tests/selenium/test_sizing.py

```diff
@@ -0,0 +1,270 @@
+import dash
+from dash.dependencies import Input, Output
+from dash.testing import wait
+from dash_table import DataTable
+from dash_html_components import Button, Div
+
+
+def cells_are_same_width(target, table):
+    wait.until(lambda: abs(target.size["width"] - table.size["width"]) <= 1, 3)
+
```

Comment:
Helper function that checks if two tables that are expected to have the same size / cell sizes are identical (or nearly identical) in size

Reply:
This routine is testing the last row of cells; while we're here would it be worthwhile checking that all the rows of a single table are the same, ie rows aren't ragged? That's certainly been a problem in the past...

Reply:
Jagged rows happen when we transition from one fragment to another of the DataTable. Checking the cells of any row of a fragment against the cells of any row of the `.cell-1-1` fragment should be sufficient if the table doesn't have a fixed portion.

What I realize from this comment is that for `szng002` I should also be testing the `target` against itself.